cmd/stack_package.go

```diff
@@ -386,18 +389,21 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 				// create a config yaml file for the tarball
 				configYaml := filepath.Join(templatePath, templates[i], ".appsody-config.yaml")
 				log.Debug.Log("configYaml is: ", configYaml)
-
 				g, err := os.Create(configYaml)
 				if err != nil {
-					return errors.Errorf("Error trying to create file: %v", err)
+					return err
 				}
-
-				// Only use major.minor version here
-				_, err = g.WriteString("stack: " + namespaceAndRepo + ":" + semver["majorminor"])
+				v := viper.New()
+				v.SetConfigFile(configYaml)
+				err = v.ReadInConfig()
 				if err != nil {
-					return errors.Errorf("Error trying to write: %v", err)
+					return err
+				}
+				v.Set("stack", namespaceAndRepo+":"+semver["majorminor"])
+				err = v.WriteConfig()
+				if err != nil {
+					return err
 				}
-
 				g.Close()
```

Comment:
Could we close the `File` from `os.Create` before reading and setting values with Viper? Just worried this might cause an error